src/sentry/grouping/strategies/base.py

```diff
@@ -29,33 +29,30 @@
 
 
 def strategy(
-    id: Optional[str] = None,
-    ids: Optional[Sequence[str]] = None,
-    interfaces: Optional[Sequence[str]] = None,
+    ids: Sequence[str],
```

Comment:
Is this function ever called with more than one id? If not, could we keep the `id=` parameter instead of `ids`, like you did with `interfaces`?

Reply:
yup for frames it is iirc